doc/source/quick_start.rst

```diff
@@ -174,11 +167,17 @@ By id
 .. code-block:: python
 
     >>> from b2sdk.v1 import DownloadDestLocalFile
+    >>> from b2sdk.v1 import SimpleProgressListener
 
     >>> local_file_path = '/home/user1/b2_example/new2.pdf'
     >>> file_id = '4_z5485a1682662eb3e60980d10_f1195145f42952533_d20190403_m130258_c002_v0001111_t0002'
     >>> download_dest = DownloadDestLocalFile(local_file_path)
+    >>> progress_listener = SimpleProgressListener('simple')
```

Comment:
do not use `SimpleProgressListener` in examples, it is only for tests

Reply:
Which one should I use then? There is a `ProgressListenerForTest` also.

Reply:
Either tqdm or the one which does nothing